packages/selection/src/useSelection.ts

```diff
@@ -211,7 +216,7 @@ export const useSelection = <Item>({
       role: role === null ? undefined : role,
```

Comment:
This can be removed now. The spread will naturally take whatever role is provided – if one is provided.